antarest/study/storage/abstract_storage_service.py

```diff
@@ -81,49 +79,15 @@ def patch_update_study_metadata(
     def get_study_information(
         self,
         study: T,
-        summary: bool = False,
     ) -> StudyMetadataDTO:
-        file_settings = {}
-        file_metadata = {}
+        try:
+            patch = Patch.parse_raw(study.additional_data.patch) or Patch()
```

Comment:
to prevent try..except performance cost here, we should always have not None data here.
Else, just make some if/elses